css/popup.css

```diff
@@ -30,23 +30,6 @@ body {
     padding: 0 15px;
 }
 
-.btn-file {
-    position: relative;
-    overflow: hidden;
-}
-
-.btn-file input[type=file] {
-    position: absolute;
-    top: 0;
-    right: 0;
-    min-width: 100%;
-    min-height: 100%;
-    font-size: 100px;
-    text-align: right;
-    filter: alpha(opacity=0);
-    opacity: 0;
-    outline: none;
-    background: white;
-    cursor: inherit;
-    display: block;
+#files {
+    display: none;
```

Comment:
![image](https://cloud.githubusercontent.com/assets/5250117/14592407/5bff7374-0548-11e6-86e9-3c5920f5cd2c.png)

@jinhduong Tested. It will not be broken. You should check the code in popup.html

https://github.com/J2TeaM/highlighter-basic/blob/master/popup.html#L24-L27